src/random_bq_to_anndata.py

```diff
@@ -1,17 +1,20 @@
 from google.cloud import bigquery
 import argparse
 import random
+import numpy as np
 import anndata as ad
-
+from scipy.sparse import csr_matrix
 
 # assumes that cas_cell_info.cas_feature_index values are a contiguous list of ints starting at 0
 def get_random_ids(project, dataset, client, num_cells):
-    query = client.query(f"SELECT MAX(cas_cell_index) AS max_table_number FROM `{project}.{dataset}.cas_cell_info`")
+    query = client.query(f"SELECT MIN(cas_cell_index) AS min_table_number, MAX(cas_cell_index) AS max_table_number FROM `{project}.{dataset}.cas_cell_info`")
```

Comment:
I must have missed this on the last review; why are these columns selected as table numbers rather than cell indexes?

Reply:
I'm beginning to wonder if we shouldn't just select the actual cell ids and randomize those... 🤔